tests/ui_tests/app/page/admin/page.py

```
@@ -489,3 +510,83 @@ def click_delete_button(self):
+            self.wait_element_visible(AdminPoliciesLocators.AddPolicyPopup.FirstStep.role_item)
+            for available_role in self.find_elements(AdminPoliciesLocators.AddPolicyPopup.FirstStep.role_item):
+                if available_role.text == role:
+                    available_role.click()
+                    break
+        assert users or groups, "There are should be users or groups in the policy"
+        if users:
+            with allure.step(f"Select users {users} in popup"):
+                self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.FirstStep.users_select)
+                self.wait_element_visible(AdminPoliciesLocators.AddPolicyPopup.FirstStep.users_item)
+                available_users = self.find_elements(AdminPoliciesLocators.AddPolicyPopup.FirstStep.users_item)
+                for user in users.split(","):
+                    for available_user in available_users:
+                        if available_user.text == user:
+                            self.scroll_to(available_user)
+                            available_user.click()
+                self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.FirstStep.users_select)
+        if groups:
+            with allure.step(f"Select groups {groups} in popup"):
+                self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.FirstStep.group_select)
+                self.wait_element_visible(AdminPoliciesLocators.AddPolicyPopup.FirstStep.group_item)
+                available_groups = self.find_elements(AdminPoliciesLocators.AddPolicyPopup.FirstStep.group_item)
+                for group in groups.split(","):
+                    for available_group in available_groups:
+                        if available_group.text == group:
+                            self.scroll_to(available_group)
+                            available_group.click()
+                self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.FirstStep.group_select)
```

Looks quite the same at the first glance. Maybe we can add a separate function that takes locators for select and item, the text value of user/group, and do stuff?
And I'm not sure what will be if the user/group isn't presented in a list. If it fail silently, it may be difficult to understand and debug

---

moved to a func